frontend/src/Student/components/ShowCard.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+import PropTypes from 'prop-types'
+import Moment from 'react-moment'
+import FaPlusCircle from 'babel-loader!react-icons/fa/plus-circle'
+import FaBook from 'babel-loader!react-icons/fa/book'
+import FaYouTube from 'babel-loader!react-icons/fa/youtube'
+import FaVimeo from 'babel-loader!react-icons/fa/vimeo'
+import { Button, ButtonGroup, Row, Col, UncontrolledTooltip } from 'reactstrap'
+
+const Card = styled.div`
+  background-color: #f8f9fa;
+  border-radius: 5px;
+  margin-top: 15px;
+  padding: 10px;
+  width: 100%;
+`
+
+const PhotoThumbnail = styled.img`
+    height: auto;
+    width: auto;
+    max-height: 10em;
+    max-width: 100%;
+`
+
+const EntryNoThumbContainer = styled.div`
+  height: auto;
+  width: auto;
+  max-height: 10em;
+  max-width: 100%;
+`
+
+const renderEntry = (props, entry) => {
```

Comment:
Should we possibly break these out into their own components? Showcard may get kind of bulky otherwise....